src/modals/DeprecationModal.jsx

```diff
@@ -0,0 +1,166 @@
+
+  const [searchParams, setSearchParams] = useSearchParams();
+  const [isModalOpen, setIsModalOpen] = useState(true);
+  const [error, setError] = useState(null);
+
+  const countryId = useCountryId();
+
+  async function handleSubmit() {
+    // Remove deprecated parameters from policy
+    const newPolicy = removeDeprecatedParams(metadata, oldPolicy);
+
+    // Create new policy with parameters given
+    const { baseline, reform } = newPolicy;
+
+    let newBaselineId = null;
+    let newReformId = null;
+
+    try {
+      if (!baseline.id) {
+        const baselineRes = await getNewPolicyId(
+          countryId,
+          baseline.data,
+          baseline.label,
+        );
+        if (!baselineRes.status === "ok") {
+          console.error("Error creating new baseline policy");
+          console.error(baselineRes);
+          setError(
+            "Error: Back-end error while trying to create new baseline policy; please try again later",
+          );
+        } else {
+          newBaselineId = baselineRes.policy_id;
+        }
+      }
+      if (!reform.id) {
+        const reformRes = await getNewPolicyId(
+          countryId,
+          reform.data,
+          reform.label,
+        );
+        if (!reformRes.status === "ok") {
+          console.error("Error creating new baseline policy");
+          console.error(reformRes);
+          setError(
+            "Error: Back-end error while trying to create new baseline policy; please try again later",
+          );
+        } else {
+          newReformId = reformRes.policy_id;
+        }
+      }
+    } catch (err) {
+      console.error("Network error while trying to create new policy");
+      console.error(err);
+      setError(
+        "Error: Network connection error while trying to create new policy; please try again later",
+      );
+    }
+
+    // On success, redirect
+    let newSearch = copySearchParams(searchParams);
+
+    // If new policy IDs are created, set them in the URL,
+    // otherwise use old; redirect to input section of calculator
+    newSearch.set("baseline", newBaselineId || baseline.id);
+    newSearch.set("reform", newReformId || reform.id);
+    newSearch.set("focus", "gov");
+    setSearchParams(newSearch);
+
+    // Destroy modal
+    setIsModalOpen(false);
+  }
+
+  function handleCloseModal() {
+    setIsModalOpen(false);
+  }
+
+  const deprecatedParamsJSX = deprecatedParams.map((param) => (
+    <p key={param} style={{ fontStyle: "italic" }}>{`- ${param}`}</p>
+  ));
+
+  return (
+    <Modal open={isModalOpen} footer={null} closable={false}>
+      <h6
+        style={{
+          paddingBottom: "16px",
+          fontWeight: 500,
+          fontSize: 20,
+        }}
+      >
+        Your policy is deprecated
+      </h6>
+      <p>
+        {`Unfortunately, as of policyengine-${countryId} v.${countryVersion}, some of your parameters are no longer supported:`}
+      </p>
+      {deprecatedParamsJSX}
```

Comment:
Could we use code-block styling for the parameters and replace the main text with label?

Reply:
We don't have access to the parameter's label at this point, as the entire item does not exist in the metadata any longer. Additionally, if we did, in some cases, the label isn't very descriptive (e.g., "7" for certain tax brackets).

Reply:
This is how this now looks.

<img width="1440" alt="Screen Shot 2024-08-12 at 7 07 32 PM" src="https://github.com/user-attachments/assets/5e8f1f82-54c5-42e4-ab9e-c3f1915f511c">

Reply:
Following a minor style edit, here's how it would look with 2+ params (the duplicate is only for visual testing):

<img width="1440" alt="Screen Shot 2024-08-12 at 7 10 58 PM" src="https://github.com/user-attachments/assets/8ea550f5-1709-4dfb-bab9-e90566926083">

Reply:
> We don't have access to the label

Oh of course lol.

Looks good- can we shade in the background like `this` though?

Reply:
This is now the look with one and with more than one param:

<img width="1440" alt="Screen Shot 2024-08-13 at 1 48 57 PM" src="https://github.com/user-attachments/assets/628de106-bc4c-4d11-b430-8ae85a6879df">
<img width="1440" alt="Screen Shot 2024-08-13 at 1 49 12 PM" src="https://github.com/user-attachments/assets/3a55652c-163b-4f47-9dba-fd1230da1863">